client/blocks/inline-help/contextual-help.js

```diff
@@ -766,329 +868,371 @@ const videosForSection = {
-			description: 'Find out how to create a website on WordPress.com in five steps.',
+			title: translate( 'Set Up a Website in 5 Steps' ),
+			description: translate( 'Find out how to create a website on WordPress.com in five steps.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/mta6Y0o7yJk',
-			title: 'Set Up a Blog in 5 Steps',
-			description: 'Find out how to create a blog on WordPress.com in five steps.',
+			title: translate( 'Set Up a Blog in 5 Steps' ),
+			description: translate( 'Find out how to create a blog on WordPress.com in five steps.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/Gx7YNX1Wk5U',
-			title: 'Create a Page',
-			description: 'Find out how to create a page on your WordPress.com site.',
+			title: translate( 'Create a Page' ),
+			description: translate( 'Find out how to create a page on your WordPress.com site.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/mCfuh5bCOwM',
-			title: 'Create a Post',
-			description: 'Find out how to create a post on WordPress.com.',
+			title: translate( 'Create a Post' ),
+			description: translate( 'Find out how to create a post on WordPress.com.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/bEVHg6nopcs',
-			title: 'Use a Custom Menu in a Widget',
-			description:
-				'Find out how to use a custom menu in a widget on your WordPress.com or Jetpack-enabled WordPress site.',
+			title: translate( 'Use a Custom Menu in a Widget' ),
+			description: translate(
+				'Find out how to use a custom menu in a widget on your WordPress.com or Jetpack-enabled WordPress site.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/nAzdUOlFoBI',
-			title: 'Configure a Static Homepage',
-			description:
-				'By default, your new WordPress.com website displays your latest posts. Find out how to create a static homepage instead.',
+			title: translate( 'Configure a Static Homepage' ),
+			description: translate(
+				'By default, your new WordPress.com website displays your latest posts. Find out how to create a static homepage instead.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/MPpVeMmDOhk',
-			title: 'Show Related Posts on Your WordPress Blog',
-			description:
-				'Find out how to show related posts on your WordPress.com site, which you can also do on a Jetpack-enabled WordPress blog.',
+			title: translate( 'Show Related Posts on Your WordPress Blog' ),
+			description: translate(
+				'Find out how to show related posts on your WordPress.com site, which you can also do on a Jetpack-enabled WordPress blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/JVnltCZUKC4',
-			title: 'Add Testimonials',
-			description: 'Find out how to add testimonials to your WordPress.com website or blog.',
+			title: translate( 'Add Testimonials' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add some testimonials' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).